50_wa_diff_overdose.py

```diff
@@ -0,0 +1,299 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import altair as alt
+from plotnine import *
+import statsmodels.api as sm
+import statsmodels.formula.api as smf
+
+# load in pre-cleaned deaths data for Washington and comparison states
+deaths = pd.read_csv("05_cleaned_data/deaths_wa.csv")
+# two separate dfs - one for washington and one for comp states
+washington = deaths[deaths["StateName"] == "Washington"]
+# comp = deaths[deaths["StateName"] != "Washington"]
```

Comment:
I am a bit unclear about why you commented out line 14. I guess that could have worked fine. The lines you wrote after 15, for creating the dataframe with comparison states would work fine, but that would have involved more typing. Line 14 was more compact and concise